### ISSUE TITLE
Upcoming work caution [DOCUMENTS]: PDF MUST be accessible

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8806]
# Upcoming work caution

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

> For the first year, provide a PDF spreadsheet link to the upcoming school year information on our tool landing page
> - Depends on Janice and Joe providing the consolidated list of new/updated school info as a spreadsheet
> - Ask them to provide this no later than first week of June, so we have at least 2 days to PDF it and add the link to the tool landing page
> - Link would be published mid-June, and we would remove the link on July 31/Aug1 -- when the 'upcoming' school info becomes 'current' in GIDS
> - ([Source](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/public-websites/yellow-ribbon-mvp/yellow-ribbon-mvp-product-brief.md#decisions))

The PDF mentioned in the Product Outline above **must** be accessible. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to access and understand the content in PDFs so I may have all the information available to any other user.

## Possible Fixes (optional)

Depending on how this PDF is generated, the process for remediating accessibility issues may differ. 
To evaluate the accessibility of a PDF:
- Open the PDF in Adobe Acrobat Pro
- Navigate to the Tools panel
- Select the Accessibility option (near the end of the list)
- Run a Full Check
- View the failed items in the panel on the left
- Remediate each item

## WCAG or Vendor Guidance (optional)

* [Section508.gov - Create Accessible PDFs](https://www.section508.gov/create/pdfs)

> 504.2.2 PDF Export. Authoring tools capable of exporting PDF files that conform to ISO 32000-1:2008 (PDF 1.7) shall also be capable of exporting PDF files that conform to ANSI/AIIM/ISO 14289-1:2016 (PDF/UA-1) (incorporated by reference, see 702.3.1). ([Source](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/text-of-the-standards-and-guidelines#E101-general))

**Comment by stephorkandatx** [comment]
@jenstrickland for this item, we are not launching the tool with a PDF. The need to make additional content available becomes something we'll need to address in summertime, There will be a transitional period in the summer (June to August) that we will want to host "upcoming" school year information while continuing to display the "current" school year info. At some point, when the "upcoming" school year becomes the "current" and data starts showing up in GIDS, we'll take down this informational asset.

@jenniferlee-dsva FYSA we've had some discussions and a walkthrough this week of what it would take to make a PDF accessible. It will be rather difficult particularly for mobile devices. Because it is temporary in nature, a PDF would be easy but presents accessibility complications. I think this may be a good content topic for the team. 

@jenstrickland with this in mind, would this remain a launchblocker or can we go to production knowing that there is no PDF now, and committing to coming up with an accessible-oriented solution post-launch and running our plans by you as soon as they're ready?

**Comment by jenstrickland** [comment]
@stephorkandatx I think the launchblocker nature would only be if the PDF was included. I can remove any defect rating on this ticket, and instead use it as a caution for the upcoming work. Sound good?

**Comment by jenstrickland** [comment]
@stephorkandatx I think this can be closed, too, since y'all opted to not do PDFs.
